src/scm/github.js

```diff
@@ -364,7 +359,7 @@ class Github {
       $('#new-gist-public').val('public');
       showAlert(`Successfully create new gist.`);
     })
-    .catch((err) => {
+    .catch(err => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/40518723e5d56dd5d2636659212a608b09d1f16a/src/scm/github.js#L362"></span>‘err’ is defined but never used.</td>
    </tr>
  </tbody>
</table>